main.go

```diff
@@ -117,11 +118,24 @@ func handle(w http.ResponseWriter, r *http.Request) {
 			return
 		} else if errors.Is(err, fs.ErrNotExist) {
 			fargs := flag.Args()
+			if len(fargs) == 0 {
+				// packages such as flag attempt to use arg0 without checking len
```

Comment:
Is this a kind of bug fix?

Reply:
I noticed when running wasmserve itself, that `flag` would panic when there were no args passed. In the case that a wasm binary attempts to use `flag`, it needs an arg0

Reply:
Also, os.Args documentation seems to indicate there should always be at least 1 value here
> Args hold the command-line arguments, starting with the program name.

Reply:
Perhaps I should use `main.wasm` rather than `.`, for consistency with `go run`

Reply:
So this fix is not related to this PR, right? Oh so can you separate the PR?

Reply:
Sure, I could separate this out.